pkg/model/model.go

```diff
@@ -136,8 +136,16 @@ func (m *Model) GetCRDs() ([]*CRD, error) {
 			crd.AddSpecField(memberNames, memberShapeRef)
 		}
 
-		// Now any additional Spec fields that are required from other API
-		// operations.
+		// A list of fields that should be processed after building the gathering
+		// the Spec and Status top level fields. The customNestedFields will be
```

Comment:
the phrase "after building the gathering the Spec and Status" feels wrong. I think there's an extra word that shouldn't be there.